Fastfile

```diff
@@ -169,7 +163,7 @@ reflected in `Info.plist` during building."
 
       desc "Get rollbar server access token from configuration file."
       rollbar_server_access_token = read_xcconfig_property(
-        xcconfig_path: project_xcconfig_path % [project_name, build_configurations[build_configuration]],
+        build_configuration: build_configurations[build_configuration],
```

Comment:
you changed the key you use here, this works the same way as before?

Reply:
Yes, the idea is to provide the `build_configurations`, and let the action build the path.